src/pages/LoginRegister/LoginPage.jsx

```diff
@@ -4,26 +4,31 @@ import { useNavigate } from "react-router-dom";
 import AuthContext from "../../context/AuthProvider";
 import LoginForm from "./LoginForm";
 import Homepage from "../Homepage";
+import { useMutation } from "../../hooks/useApi";
 export default function LoginPage() {
   const navigate = useNavigate();
-
+  const userLogin = useMutation("/users/login", {
+    method: "post",
+  });
   const { userDetails, login, authorized } = useContext(AuthContext);
 
-  const LoginFormDetail = (details) => {
-    const founduser = userDetails.find((user) => user.email === details.email);
-    if (founduser) {
-      if (details.password === founduser.password) {
-        console.log("loged in");
-        alert("Logged in!");
-        login();
-        navigate("/homepage");
-      } else {
-        console.log("Your password is wrong!");
-        alert("Your password is wrong!");
-      }
-    } else {
-      console.log("No such account, Please register first!");
-      alert("No such account, Please register first!");
+  const LoginFormDetail = async (details) => {
+    const loginDetails = {
+      data: {
+        email: details.email,
+        plaintextPassword: details.password,
+      },
+    };
```

Comment:
```suggestion
  const LoginFormDetail = async ({email, password: plaintextPassword}) => {
    const loginDetails = {
      data: {
        email,
        plaintextPassword,
      },
    };
```

Check this works first :P